src/System.Private.ServiceModel/src/System/ServiceModel/Channels/MessageContent.cs

```diff
@@ -21,23 +22,21 @@ internal abstract class MessageContent : HttpContent
         protected MessageEncoder _messageEncoder;
         protected Stream _stream = null;
         private bool _disposed;
+        protected TaskCompletionSource<bool> _writeCompletedTcs;
```

Comment:
Agreed. Again, I think this should only be applied to master because of AskMode timing.